src/main/java/mate/academy/service/CinemaHallService.java

```diff
@@ -7,6 +7,10 @@ public interface CinemaHallService {
     CinemaHall add(CinemaHall cinemaHall);
 
     CinemaHall get(Long id);
-    
+
     List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
```

Comment:
Consider changing the return type of the `update` method to `void` to better reflect the operation's outcome. Returning a `Boolean` might not be necessary if the method is expected to always succeed or throw an exception otherwise.